runner_class.py

```diff
@@ -0,0 +1,36 @@
+import asyncio
+
+
+class Runner:
+    def __init__(self, name, step_per_sec, start_point, do_after_step=None):
+        self.name = name
+        self.step_per_sec = step_per_sec
+        self.start_point = start_point
+        self.do_after_step = do_after_step
+        self.step = 0
+
+    async def run(self, remaining_distance: int, step_callback, finish_callback):
+        if remaining_distance == 0:
+            finish_callback(self)
+            return self
+
+        await asyncio.sleep(1 / self.step_per_sec)
+        self.step += 1
+        step_callback()
+        if self.do_after_step is not None:
```

Comment:
if self.do_after_step 
is equal to 
if self.do_after_step is not None:

Use the first one, its shorter :)